examples/resources/powerstore_host/host.tf

```diff
@@ -0,0 +1,11 @@
+# Commands to run this tf file : terraform init && terraform plan && terraform apply
+# Create, Update, Delete is supported for this resource
+# To import , check host_import.tf for more info
+# name,os_type and initiators are the required attributes to create and update
```

Comment:
below line 4, can you add which are the optional one attributes ??

Reply:
Added